contracts/HATVaults.sol

```diff
@@ -12,54 +12,104 @@ import "./vaults/Getters.sol";
 import "./vaults/Withdraw.sol";
 
 // Errors:
-// HVE01: Only committee
-// HVE02: Active claim exists
-// HVE03: Safety period
-// HVE04: Beneficiary is zero
-// HVE05: Not safety period
-// HVE06: reward percentage is higher than the max bounty
-// HVE07: Withdraw request pending period must be <= 3 months
-// HVE08: Withdraw request enabled period must be >= 6 hour
-// HVE09: Only callable by governance or after 5 weeks
-// HVE10: No active claim exists
-// HVE11: Amount to reward is too big
-// HVE12: Withdraw period must be >= 1 hour
-// HVE13: Safety period must be <= 6 hours
-// HVE14: Not enough fee paid
-// HVE15: Vesting duration is too long
-// HVE16: Vesting periods cannot be zero
-// HVE17: Vesting duration smaller than periods
-// HVE18: Delay is too short
-// HVE19: No pending max bounty
-// HVE20: Delay period for setting max bounty had not passed
-// HVE21: Committee is zero
-// HVE22: Committee already checked in
-// HVE23: Pool does not exist
-// HVE24: Amount to swap is zero
-// HVE25: Pending withdraw request exist
-// HVE26: Deposit paused
-// HVE27: Amount less than 1e6
-// HVE28: Pool balance is zero
-// HVE29: Total split % should be `HUNDRED_PERCENT`
-// HVE30: Withdraw request is invalid
-// HVE31: Token approve failed
-// HVE32: Wrong amount received
-// HVE33: Max bounty cannot be more than `HUNDRED_PERCENT`
-// HVE34: LP token is zero
-// HVE35: Only fee setter
-// HVE36: Fee must be less than or equal to 2%
-// HVE37: Token approve reset failed
-// HVE38: Pool must not be initialized
-// HVE39: Set shares arrays must have same length
-// HVE40: Committee not checked in yet
-// HVE41: Not enough user balance
-// HVE42: User shares must be greater than 0
-// HVE43: Swap was not successful
-// HVE44: Routing contract must be whitelisted
-// HVE45: Not enough HATs for swap
-// HVE46: Not enough rewards to transfer to user
-// HVE47: Bounty split must include hacker payout
-// HVE48: Claim can only be approved if challengeperiod is over, or if the caller is the arbitrator
+// Only committee
+error OnlyCommittee();
+// Active claim exists
+error ActiveClaimExists();
```

Comment:
Shouldn't all the errors be declared on the Base? I'm surprised it compiles like this actually :)

Reply:
yeah @lirona :P